usingcurl/reademail.md

```diff
@@ -21,21 +21,77 @@ To delete message 1:
 
 ## IMAP
 
-Get the mail using the UID 57 from mailbox 'stuff':
+For clarity, server authentication with `--username` / `-u` is elided, and
+(while usually not required) IMAP commands and their arguments are upper-cased.
+For reasons noted below, you will probably want to include curl's `-v` or
+`--verbose` option when initially tinkering with its IMAP support.
 
-    curl imap://server.example.com/stuff;UID=57
+List (sub)mailboxes in the main inbox:
 
-Instead, get the mail with index 57 from the mailbox 'fun':
+    curl imap://server.example.com
 
-    curl imap://server.example.com/fun;MAILINDEX=57
+List (sub)mailboxes in the mailbox `boring`, with the IMAP server on a
+non-standard port number:
 
-List the mails in the mailbox 'boring':
+    curl imap://server.example.com:1143/boring
 
-    curl imap://server.example.com/boring
+List all emails in the inbox:
 
-List the mails in the mailbox 'boring' and provide user and password:
+    # use quotes appropriate to your shell here to preserve internal whitespace
+    curl imap://server.example.com:1143/INBOX -X 'FETCH 1:* FAST'
 
-    curl imap://server.example.com/boring -u user:password
+This produces output similar to what's below; the numbers after the asterisks
+are the **mailbox indices** of the messages, counting from 1:
+
+    * 1 FETCH (FLAGS (\Flagged \Deleted \Seen) INTERNALDATE " 3-Dec-2025 12:02:39 -0500" RFC822.SIZE 15163)
```

Comment:
Output from a command. Not sure I would want it wrapped just to appease [`check-quotes.pl`](https://github.com/curl/everything-curl/blob/09c52879877f4540a66fa56a929555e80f8307b0/.github/scripts/check-quotes.pl#L33), but if this is a hard choice (_e.g._ because it makes the PDF look bad, or we don't want to alter the CI routines around this), I'll just remove it.